include/tree_walker.h

```diff
@@ -0,0 +1,134 @@
+#pragma once
```

Comment:
Could you add a file header above this line. You can just copy-paste the block comment from here: https://github.com/makaimann/smt-switch/blob/68a0a4180aefb4de756178c7a42b79d73db1243e/include/identity_walker.h#L2. Then replace the filename, \brief description, and put your name there.